cmds/cmds.go

```diff
@@ -0,0 +1,82 @@
+package cmds
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/pkg/browser"
+	"github.com/riraum/hn-cli/item"
+)
+
+func OpenLink(URL string) error {
+	if err := browser.OpenURL(URL); err != nil {
+		return fmt.Errorf("Failed to open `%s`: %w", URL, err)
+	}
+
+	return nil
+}
+
+func Cmds(input []string, postUnmarsh item.Item) (string, error) {
```

Comment:
can you explain why the second parameter is called `postUnmarsh` and not `i` or `post`?

```suggestion
func Cmds(input []string, postUnmarsh item.Item) (string, error) {
```

Reply:
I just used the existing naming, but I get that this doesn't make too much sense now. Will change.